thrifty-schema/src/test/java/com/microsoft/thrifty/schema/parser/ThriftParserTest.java

```diff
@@ -1124,6 +1124,45 @@ public void trailingDocWithoutSeparatorWithAnnotationOnNewLine() {
     }
 
     @Test
+    public void enumJavadocWithoutSeparators() {
+        String thrift = "/**\n" +
+                " * Some Javadoc\n" +
+                " */\n" +
+                "enum Value {\n" +
```

Comment:
Yes, it did - this is basically the snippet you shared, minus one member and with things renamed.  Not sure why it wouldn't have failed for you.